docs/spec_draft.md

```diff
@@ -1406,6 +1407,76 @@ and produces a `result` tensor. More formally,
 ```
 
 [Back to Ops](#index-of-ops)
+
+## stablehlo.rng
+
+### Semantics
+
+Constructs an output of a given shape with random numbers generated given
+`rng_distribution`.
+
+`rng_distribution` is one of the following:
+  * `UNIFORM`: the uniform distribution over the interval `[a,b)`. The
+    parameters and output element type have to be a boolean type, an integral
+    type or a floating point types, and the types have to be consistent.
+  * `NORMAL`: the normal distribution with parameters $\mu$ = `a` and $\sigma$ =
+    `b`. The parameters and output shape have to have a floating point
```

Comment:
The parameter type and shape are already clarified in Inputs/Outputs and Constraints. SPO we can skip this "The parameters ...scalar valued."